src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Layouts/Spacer/BitSpacerDemo.razor.scss

```
@@ -1,5 +1,64 @@
+// The MinGap example only tells its story while the container is being squeezed, so the reader
+// gets a native resize handle to squeeze it with.
+.demo-resizable {
+    resize: horizontal;
+    overflow: auto;
+    max-width: 100%;
+    min-width: 10rem;
+}
+
+// The element is rendered by BitSpacer itself, so it carries its own scope and only ::deep rules
+// can reach the classes handed to it through Class.
+::deep .custom-class {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)